src/main/java/com/spotify/docker/client/messages/HostConfig.java

```diff
@@ -1036,7 +1038,12 @@ public String toString() {
       } else if (from == null || from.equals("")) {
         return to;
       } else if (readOnly == null || !readOnly) {
-        return from + ":" + to;
+          if (noCopy == null || !noCopy) {
```

Comment:
Phew! This option is cursed, it is supported in bind mounts although the man page says it so :

https://github.com/docker/docker/issues/27046